lib/api/model.js

```diff
@@ -15,6 +17,101 @@ function get (modelName, passByReference) {
     return clone(linz.get('models')[modelName]);
 }
 
+/**
+ * Get permission data for a model
+ * @param {Object}   req      Request data
+ * @param {String}   modelName Model name
+ * @param {Function} cb        Callback function
+ * @return {Object}
+ */
+function getPermissions(req, modelName, cb) {
+
+    if (!linz.get('disable permission cache') && !req.linz.cache.permissions.models.invalidate && req.session.permissions && req.session.permissions[modelName]) {
+        return cb(null, req.session.permissions[modelName]);
+    }
+
```

Comment:
Could we please have some debug output here saying `Priming models cache.` The debug should use the `debug` module, with a setting of `linz:cache`. That will allow us to get a handle of when the cache is/isn't being regenerated.